android-components/components/ui/icons/src/main/res/values/colors.xml

```diff
@@ -9,4 +9,7 @@
     <!-- Star icon fill colors for mozac_ic_star_one_half_fill_20 -->
     <color name="mozac_ic_star_filled">#000000</color>
     <color name="mozac_ic_star_unfilled">#FFFFFF</color>
+
+    <!-- Private Mode mask icon fill colors for mozac_ic_private_mode_circle_fill_20 -->
+    <color name="mozac_ui_mask_icon_fill">#ff952bb9</color>
```

Comment:
Instead of hardcoding the color in ui-icons, we actually want to do this at the app level. That way you will be able to use the photon color or design color variable (if one exist for such use case). For the color here, I would suggest using a generic black or white (eg `#000000`).

See: 
https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/colors.xml#361
https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values-night/colors.xml#173

You would set photonPurple60 here in both the fenix Light and Dark `colors.xml`. I did add a comment on the figma that we currently using a iOS color in the Android design which we might want to follow up on when @topotropic is back  from PTO https://www.figma.com/file/Ha2T4I0nX0h7B95GYxH6wV?node-id=4188:9468&mode=design#576794614.

Reply:
implemented the colors in the `res` files like you suggested, this will be easier to chase down than going back and modifying A-C again when we need to update the colors.